alex/applications/PublicTransportInfoCS/directions.py

```diff
@@ -505,11 +519,24 @@ def search_train_station(self, stop_mask, max_count=0, skip_count=0):
             max_count, # max count
             REG.SMART, # return regions
             skip_count, # skip count
-            TTINFODETAILS.STATIONSEXT,
-            COOR.DEFAULT,
+            #TTINFODETAILS.STATIONSEXT,
```

Comment:
are you sure you don't need these parameters? i think they should be replaced by zeros in that case... otherwise `TTLANG.ENGLISH` is actually passed as a constant for `TTINFODETAILS` (the parameters are positional)